### PR TITLE
Knox, Bootstrap and design rework

### PR DESCRIPTION
**Pull request by DFilipeS**
Removed JWT and added Django REST Knox. Removed Foundation and added Bootstrap. Reworked design.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ We build on the solders of giants with the following technologies:
 
 * [React](https://github.com/facebook/react)
 * [React Router](https://github.com/rackt/react-router)
-* [JSON Web Token](https://www.npmjs.com/package/jsonwebtoken) JSON Web Token for API authentication
+* [Django REST Knox](https://github.com/James1345/django-rest-knox) Token based authentication for API endpoints
```
Comment (pedrorodriguesgomes) [inline README.md]: this should be added to backend section

File: package.json [hunk 2]
```diff
@@ -32,6 +32,8 @@
     "babel-preset-stage-1": "6.13.0",
```
Comment (lfrodrigues) [inline package.json]: you should update all packages to most recent versions

File: py-requirements/base.txt [hunk 3]
```diff
@@ -23,3 +23,4 @@ python-dateutil==2.5.3
 # Logging
```
Comment (lfrodrigues) [inline py-requirements/base.txt]: remove old unused packages, update packages

File: webpack/prod.config.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ module.exports = {
 
     entry: {
```
Comment (lfrodrigues) [inline webpack/prod.config.js]: make sure css is getting minified in prod config

File: src/static/containers/Login/index.js [hunk 5]
```diff
@@ -67,54 +67,36 @@ class LoginView extends React.Component {
...
-                <div className="row margin-top-large">
-                    <div className="small-6 small-centered columns">
-                        <div className="row">
-                            <div className="small-12 columns">
-                                <h1>Log in to view protected content!</h1>
-                            </div>
+            <div className="container login">
+                <h1 className="text-center">Login</h1>
+                <div className="login-container margin-top-medium">
+                    <form>
```
Comment (lfrodrigues) [inline src/static/containers/Login/index.js]: Form should be changed to tcomb

**Comment by DFilipeS** [comment]
Fixed, please check my latest changes.

File: README.md [hunk 6]
```diff
@@ -8,18 +8,18 @@ We build on the solders of giants with the following technologies:
...
 * [Redux](https://github.com/rackt/redux)'s futuristic [Flux](https://facebook.github.io/react/blog/2014/05/06/flux.html) implementation
 * [Redux Dev Tools](https://github.com/rackt/redux-devtools) for next generation DX (developer experience). Watch [Dan Abramov's talk](https://www.youtube.com/watch?v=xsSnOQynTHs)
 * [Redux Thunk](https://github.com/gaearon/redux-thunk) Thunk middleware for Redux - used in async actions
 * [React Router Redux](https://github.com/rackt/react-router-redux) Ruthlessly simple bindings to keep react-router and redux in sync
 * [fetch](https://github.com/github/fetch) A window.fetch JavaScript polyfill
 * [style-loader](https://github.com/webpack/style-loader), [sass-loader](https://github.com/jtangelder/sass-loader) and [less-loader](https://github.com/webpack/less-loader) to allow import of stylesheets in plain css, sass and less,
 * [font-awesome-webpack](https://github.com/gowravshekar/font-awesome-webpack) to customize FontAwesome
-* [foundation-sites](https://github.com/zurb/foundation-sites) for the grid
+* [bootstrap-sass-loader](https://github.com/shakacode/bootstrap-sass-loader) to customize Bootstrap
```
Comment (lfrodrigues) [inline README.md]: I guess git is wrong, see deprecation message
https://github.com/shakacode/bootstrap-sass-loader

**Pull request merged**